mvp-spec/workflows.md

```diff
@@ -0,0 +1,84 @@
+# Workflows
+
+The workflows part of the Prebid Addressability Framework are:
+- User Id and Preferences
+- Selling ad slots
+
+## Id and User Preferences
+
+### Generic workflow
+
+This workflow illustrates retrieving and creating/setting User Id and Preferences.
```

Comment:
I'm a little concerned that there is just one workflow that illustrates the visit of an unknown user.
To my mind, there are two relatively different workflows:
- first visit (unknown user) and creation of id and preferences
- visit from an existing user

I suppose it's ok **to only show the first case** (creation), but then, I wouldn't claim it's "retrieving and creating/setting", it's just "creating/setting".

Reply:
Agreed. New commit added with three diagrams: creating, retrieving, updating User Id and Preferences.

Please let me know if these diagrams are correct, in particular how the Operator is on the critical path (or not) of U & P update.